woocommerce/woocommerce-android
# Highlight the selected order in the order list

Closes: #10595 


### Description

This PR highlights selected PR to visually distinguish the chosen order within the order list, specifically within a two-pane layout configuration. The highlighted order persists across pull-to-refresh, navigation across different tabs.

### Testing instructions

1. Open order listing screen in tablet device
2. Select any order item from the order list
3. Ensure the selected order is highlighted
4. Pull-To-Refresh and ensure the selected order item is still highlighted
5. Navigate across different tabs, come back to the order list and see the previously selected order item is still highlighted and it's corresponding screen is opened in detail pane

### Images/gif

[highlight_select_order.webm](https://github.com/woocommerce/woocommerce-android/assets/1331230/8e81ab8f-ea7e-4332-9297-a5eac6dac01c)
